docs/en/reference/transactions.rst

```diff
@@ -48,27 +60,46 @@ constants:
     TransactionIsolationLevel::SERIALIZABLE
 
 The default transaction isolation level of a
-``Doctrine\DBAL\Connection`` is chosen by the underlying platform
-but it is always at least ``READ_COMMITTED``.
+``Doctrine\DBAL\Connection`` instance is chosen by the underlying
+platform but it is always at least ``READ_COMMITTED``.
+
+Emulated Transaction Nesting
```

Comment:
I don't believe this mode deserves to be called an emulation. Emulation implies that the end result is achieved through some indirect means. But this mode doesn't achieve. It's effectively a null implementation. Not sure how to name it better though.

Reply:
I describe both modes under this section, but I see what you mean. Let's call the first mode "Fallback behavior"?

Reply:
I'd call it a dummy mode maybe? A "fallback" implies that it's not the default behavior, and falling back still achieves the desired result (at least partially). But here it's the default mode and it only looks like nesting while it's not by any means.

Reply:
Ok let's use that, I don't have a better suggestion right now.